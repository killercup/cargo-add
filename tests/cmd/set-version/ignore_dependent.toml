bin.name = "cargo-set-version"
args = ["set-version", "2.0.0", "-p", "cargo-list-test-fixture"]
status = "success"
fs.sandbox = true

[env.add]
CARGO_IS_TEST="1"
