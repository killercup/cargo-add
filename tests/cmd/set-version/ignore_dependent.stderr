    Upgraded cargo-list-test-fixture from 0.0.0 to 2.0.0
